v1/charm-dev-batteries.yaml

```diff
@@ -5,7 +5,7 @@
 # If you are a zsh user, the ohmyzsh juju plugin is already enabled when you switch to zsh.
 #
 # To create a VM similar to a GitHub-hosted runner:
-# multipass launch --mem 7G --cpus 2 --name charm-dev-2cpu-7g charm-dev
+# multipass launch --mem 7G --cpus 2 --name charm-dev-2cpu-7g charm-dev-batteries
```

Comment:
There is another problem here. One cannot use `--cpus 2`, because the minimun CPU count prescribed in the definition is 3. Is the 2 or the 3 wrong? :P

Reply:
This command is to create a VM similar to a GitHub-hosted runner. Take a look at line 7

Reply:
Yes, but executing it will give:
```
$ multipass launch --mem 7G --cpus 2 --name charm-dev-2cpu-7g charm-dev-batteries
launch failed: Requested Number of CPUs is less than Blueprint minimum of 3
```

Reply:
Agreed.